v2/datastream-to-bigquery/src/main/java/com/google/cloud/teleport/v2/templates/DataStreamToBigQuery.java

```diff
@@ -422,6 +419,10 @@ public void process(
     return pipeline.run();
   }
 
+  private static Set<String> getFieldsToIgnore(String fields) {
+    return Arrays.stream(fields.trim().split("\\s*,\\s*")).collect(Collectors.toSet());
```

Comment:
Could you switch this to use [Guava Splitter](https://guava.dev/releases/23.0/api/docs/com/google/common/base/Splitter.html#on-java.util.regex.Pattern-)? I'm pretty sure you can use it without any changes to the POM.